subcommittees/mentoring/PracticumProposal.md

```diff
@@ -0,0 +1,192 @@
+- Time zone plus preferred time of day for meetings (daytime/evening) 
+
+Mentors will be selected and matched to groups by the mentoring committee.  
+
+Before starting to meet with their groups, all mentors are required to attend the mentoring training
+meeting. 
+
+### Lead mentor
+
+There will be a lead mentor who will lead the mentors in their work. That person will be in charge of
+mentor training, and for keeping track of the progress of the groups by staying in touch with the
+mentors. 
+
+### Mentoring
+
+Mentors will go through a one hour training meeting which will include an overview of responsibilities,
+materials for facilitating discussions with their groups, methods of giving and receiving feedback, and
+a review of the code of conduct.
+
+It is the mentor’s responsibility to email their group members and set up a time to meet and an
+expected lesson schedule (this can be over email or via an initial meeting).  
+
+Each week, the group meets without the mentor to discuss their experience with the lesson.  The group
+also meets with their mentor once a week for a more in-depth discussion.  Depending on the number of
+groups and mentors, one mentor may be responsible for two groups.  We suggest that students meet on
+Tues/Wed, with mentor on Friday or weekend, if convenient for the group. 
+
+### Mini-lessons
+
+The following is the proposed mini-lesson plan:  
+
+1. Trainees will in collaboration with their mentor figure out a portion of material that the student
+will teach from. The aim is to have approximately an hours worth of material. The trainee will then
+prepare the material, including data, and make it available for review by the mentor. 
+2. The mentor will then review the lesson plan and have a short (online) meeting with each trainee one
+week before the trainee is scheduled to teach their Mini-lesson. Guidance on what to teach and how to
+teach it will be given both via email and during face-to-face communication. 
+3. The trainee teaches the 1 hr Mini-lesson, in person, to a minimum of 1 other individual while the
+mentor observes either in person, or remotely. 
+4. Directly after the Mini-lesson, the trainee and mentor engage in a debrief where the mentor first
+asks the trainee to comment on their performance before they give feedback on the trainee’s performance. 
+5. Within a week or two after the trainee has completed teaching their Mini-lesson, the mentor and
+trainee will meet together for a debrief. The trainee will before this session fill out a short
+questionnaire about how their lesson went, which will feed into the discussion with their mentor.
+Questionnaire will be provided by the SCF. This debrief will preferably happen in groups, as the mentor
+is likely to have several trainees that have taught the same material for their Mini-lessons. 
+6. Trainees will be encouraged to make a suggested revision/edit/addition (e.g. new challenge question)
+to the lesson topic they taught.  
+
+## Timeline
```

Comment:
I would rather not. Those are implementation details, which locks this one down hard.